packages/dx-react-grid/docs/reference/table-selection.md

```diff
@@ -13,48 +13,40 @@ This plugin visualizes the selection state within a table by rendering selection
 
 Name | Type | Default | Description
 -----|------|---------|------------
-highlightSelected | boolean | false | If true, selected rows are highlighted. Note that the `Table` plugin's `rowComponent` is ignored in this case.
-selectByRowClick | boolean | false | If true, a selected row is toggled by click. Note that the `Table` plugin's `rowComponent` is ignored in this case.
-showSelectAll | boolean | true | If true, the 'select all' checkbox is rendered inside the heading row.
-showSelectionColumn | boolean | false | If true, selection checkboxes are rendered inside each data row.
-selectCellTemplate | (args: [SelectCellArgs](#select-cell-args)) => ReactElement | | A component that renders a data row selection checkbox.
-selectAllCellTemplate | (args: [SelectAllCellArgs](#select-all-cell-args)) => ReactElement | | A component that renders the Select All checkbox.
+highlightRow | boolean | false | If true, selected rows are highlighted. Note that `Table` plugin's `rowComponent` is ignored in this case.
+selectByRowClick | boolean | false | If true, the row's selection state is toggled when a user clicks the row. Note that `Table` plugin's `rowComponent` is ignored in this case.
+showSelectAll | boolean | true | If true, the 'select all' checkbox is rendered in the header row.
+showSelectionColumn | boolean | true | If true, the selection column (that displays a selection checkboxes) is rendered.
+cellComponent | ElementType&lt;[TableSelectCellProps](#tableselectcellprops)&gt; | | A component that renders a selection cell (a cell containing a selection checkbox).
+headerCellComponent | ElementType&lt;[TableSelectHeaderellProps](#tableselectcellprops)&gt; | | A component that renders a cell containing the 'Select All' checkbox.
```

Comment:
It looks like this anchor `#tableselectcellprops` is wrong. Should be `#tableselectheadercellprops`.

Reply:
TableSelectHeaderellProps -> TableSelectHeader**C**ellProps